cylc/uiserver/tests/test_schema.py

```diff
@@ -30,3 +31,19 @@ def test_node_map():
         _class = getattr(cylc.uiserver.schema, type_name)
         # It is not straightforward to check that _class is a Graphene class
         assert 'graphene' in type(_class).__module__
+
+
+async def test_mutator():
+    """Test exception and arg variants."""
+    class Info:
+        context = {'resolvers': None, 'id': 1}
+
+    with pytest.raises(Exception) as exc:
+        await mutator(
+            None,
+            Info(),
+            command='NotACommand',
+            workflows=None,
+            args={'This': 'That'}
+        )
+    assert 'NoneType' in f'{exc.value}'
```

Comment:
This doesn't seem right to me. It looks like you're trying to test that `NotACommand` is handled properly? But the exception is `AttributeError: 'NoneType' object has no attribute 'log'` which is unrelated (due the line above `context = {'resolvers': None,`)

Reply:
Yeah, it was just a cheap way of hitting most the code without creating a resolver object.. But I suppose I could